shared/src/main/scala/mlscript/NewParser.scala

```diff
@@ -648,7 +648,7 @@ abstract class NewParser(origin: Origin, tokens: Ls[Stroken -> Loc], newDefs: Bo
         exprCont(Super().withLoc(S(l0)), prec, allowNewlines = false)
       case (IDENT("?", true), l0) :: _ => 
```

Comment:
Why do you need a special case for this?

Reply:
removing it will cause unexpected operator in parsing